ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1,4 +1,49 @@
 periodics:
+- agent: kubernetes
+  #labels:
+  #  ci.openshift.io/role: infra
+  cluster: ci/api-build01-ci-devcluster-openshift-com:6443
```

Comment:
has to run there because it will `oc apply` to update the secret there.

Or we supple the kubeconfig file to job. This might be a better idea.

Reply:
I don't want the build farm to have any secret data that isn't needed.

Reply:
This is what i meant by secret:

https://github.com/openshift/release/blob/908dd037fd8aa7ef24b2b9ddaa5623973fd35315/ci-operator/jobs/infra-periodics.yaml#L24

The `secret/app-cert` on build01 actually holds the CA certifates for the cluster.

We are using the incluter config right now. That is why this job has to run on build01.
If we mount a kubeconfig file, we could run it on any cluster.

Reply:
Done.